azureeyemodule/app/streaming/rtsp.hpp

```diff
@@ -49,6 +49,7 @@ void set_stream_params(const StreamType &type, const std::string &resolution);
 void set_stream_params(const StreamType &type, int fps, bool enable);
 void set_stream_params(const StreamType &type, const std::string &resolution, bool enable);
 void set_stream_params(const StreamType &type, const std::string &resolution, int fps, bool enable);
+void take_snapshot(const StreamType &type);
```

Comment:
Please document this function like all the other ones in the header file. The set_stream_params all have the same documentation (that's why most of them don't have any).